lambdas/initialise-ipv-session/src/main/java/uk/gov/di/ipv/core/initialiseipvsession/InitialiseIpvSessionHandler.java

```diff
@@ -411,6 +415,29 @@ private boolean isHmrcInheritedIdentityWithStrongerVotPresent(
         }
     }
 
+    private void sendInheritedIdentityReceivedAuditEvent(
+            SignedJWT signedInheritedIdentityJWT, AuditEventUser auditEventUser)
+            throws SqsException, ParseException, AuditExtensionException, CredentialParseException {
+        try {
+            Vot vot = userIdentityService.getVot(signedInheritedIdentityJWT);
+            var claimsSet = signedInheritedIdentityJWT.getJWTClaimsSet();
+            var vc = (JSONObject) claimsSet.getClaim(VC_CLAIM);
+            var evidence = vc.getAsString(EVIDENCE);
+            AuditExtensionsVcEvidence auditExtensions =
+                    new AuditExtensionsVcEvidence(claimsSet.getIssuer(), evidence, null, vot);
```

Comment:
The only thing we could consider is whether it's worth using a shared helper for this logic - we've got at least three copies of it now:
- `AuditCriResponseHelper#getExtensionsForAudit`
- `CriStoringService#getAuditExtensions`
- This method

I wonder if we should: rename `AuditCriResponseHelper` to `AuditExtensionsHelper` and move into the common `audit-service` lib, and then update all usages to use that.

For extra credit, we could consider moving some of the methods from `VcHelper` there. You'll need to rebase/merge to see those changes though.